### ISSUE TITLE
links.rst file update?

### ISSUE THREAD

**Issue by PipKat** [pyansys/pyansys-math#86]
### Description of the modifications

Line 36 has this link: . _pyansys_support: pyansys.support@ansys.com

I'm not sure if it's even used, but we might want to change it to pyansys.core@ansys.com.

**Comment by clatapie** [comment]
You're right, thank you for pointing this out. We should remove the first email mention.
